soroban-env-host/src/host/conversion.rs

```diff
@@ -172,10 +172,14 @@ impl Host {
         })
     }
 
+    pub(crate) fn sha256_hash_from_bytes(&self, bytes: &[u8]) -> Result<Vec<u8>, HostError> {
```

Comment:
I'll file a followup issue to clean it up. I agree this function is just "compute" instead of conversion. We don't currently have a `crypto.rs` and adding it for this one function seems not worth it. I have a few other tasks of adding other primitives and will address this when that comes.

Reply:
Followup issue is good, thanks.